presto-benchmark/src/test/java/com/facebook/presto/benchmark/MemoryLocalQueryRunner.java

```diff
@@ -35,12 +42,13 @@
 
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
 import static io.airlift.units.DataSize.Unit.GIGABYTE;
+import static java.util.Objects.requireNonNull;
 
 public class MemoryLocalQueryRunner
 {
     protected LocalQueryRunner localQueryRunner = createMemoryLocalQueryRunner();
 
-    public void execute(@Language("SQL") String query)
+    public List<Page> execute(@Language("SQL") String query)
```

Comment:
I think this commit is out of order.  It is at the end, but I think `BenchmarkInequalityJoin` uses the result.

Also, I think the other benchmarks that use this connector might be broken since they don't return the results, and the JVM might just eliminate the code.